app/src/main/java/org/mozilla/focus/MainActivity.kt

```diff
@@ -189,9 +189,11 @@ class MainActivity : LocaleAwareAppCompatActivity(), OnUrlEnteredListener, Media
                 null, null)
     }
 
-    override fun onTextInputUrlEntered(urlStr: String,
-                                       autocompleteResult: InlineAutocompleteEditText.AutocompleteResult?,
-                                       inputLocation: UrlTextInputLocation?) {
```

Comment:
I've never liked this style of indent so I'm glad ktlint has made them impossible. Yeah!